### ISSUE TITLE
Program Crashes With Error: Cannot unmarshal object into Go struct field 

### ISSUE THREAD

**Issue by iworkinpixels** [erroneousboat/slack-term#271]
Version: 0.5.0
Installation method: Direct download of binary (M1 Mac running Monterey)

On visiting any channels with incompatible blocks, the program crashes with the following error message:

`cannot unmarshal object into Go struct field Attachment.messages.attachments.blocks of type slack.Block`

**Comment by rahmerh** [comment]
Can confirm, this is basically making this unusable for me at the moment.

Installed using go install.

`2022/08/20 20:39:37 json: cannot unmarshal object into Go struct field Attachment.messages.attachments.blocks of type slack.Block`

Are there any updates for this?

**Comment by robwithhair** [comment]
Yeah, would be good to note on readme that app isn't currently working to avoid wasted install time.  Installed via AUR binary build.

**Comment by tkasci** [comment]
Still not working.
